Apeswap-Bots/APESWAP-BOT-2/src/agent.spec.ts

```diff
@@ -0,0 +1,78 @@
+import { Finding, HandleTransaction, TransactionEvent } from "forta-agent";
+import { tokenType, includeAndExcludeFunctions, createFinding, provideBotHandler } from "./agent";
```

Comment:
This import should probably be avoided, try mocking everything (e.g. TEST_INCLUDE_AND_EXCLUDE_FUNCTION, testCreateFinding etc) in the `.spec.ts` file.